packages/deployer/subtasks/print-info.js

```diff
@@ -11,12 +11,6 @@ const { readPackageJson } = require('@synthetixio/core-js/utils/npm');
 const { SUBTASK_PRINT_INFO } = require('../task-names');
 
 subtask(SUBTASK_PRINT_INFO, 'Prints info about a deployment.').setAction(async (taskArguments) => {
-  const { quiet } = taskArguments;
-
-  if (quiet) {
-    return;
-  }
-
```

Comment:
Removing this we still execute the logger so is covered on coverage, but nevertheless, it doesn't prints nothing because is configured on silent mode at the beggining of the deploy task.